Sources/HummingbirdMustache/Library+FileSystem.swift

```diff
@@ -16,27 +16,29 @@ import Foundation
 
 extension HBMustacheLibrary {
     /// Load templates from a folder
-    func loadTemplates(from directory: String, withExtension extension: String = "mustache") throws {
+    static func loadTemplates(from directory: String, withExtension extension: String = "mustache") throws -> [String: HBMustacheTemplate] {
```

Comment:
Consideration: Do we plan to load templates using NIOFileSystem?

Reply:
We can move to that, but we need NIOFileSystem to be properly released.

Reply:
Agreed. Though we should then prepare by ensuring all public API is `async` in case it touches the FileSystem